pkg/controller/pxc/backup.go

```diff
@@ -73,73 +74,58 @@ func (r *ReconcilePerconaXtraDBCluster) reconcileBackups(cr *api.PerconaXtraDBCl
 				return fmt.Errorf("storage %s doesn't exist", bcp.StorageName)
```

Comment:
Should all backups be failed because one has the wrong storage?

Reply:
81a9c2fd